backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -323,8 +323,10 @@ object BackendSettings extends BackendSettingsApi {
             case _ =>
           }
           windowExpression.windowFunction match {
-            case _: RowNumber | _: AggregateExpression | _: Rank | _: CumeDist | _: DenseRank |
-                _: PercentRank | _: NthValue | _: NTile | _: Lag | _: Lead =>
+            case _: RowNumber | _: Rank | _: CumeDist | _: DenseRank | _: PercentRank |
+                _: NthValue | _: NTile | _: Lag | _: Lead =>
+            case aggrExpr: AggregateExpression
+                if !aggrExpr.aggregateFunction.isInstanceOf[ApproximatePercentile] =>
```

Comment:
cc: @zhztheplayer @JkSelf

Reply:
So this is because Velox doesn't yet support approximate percentile. Perhaps after https://github.com/apache/incubator-gluten/pull/5007/files#diff-dc93325490d76d420bdb064320fd3e79da8a05c20eea0298c5638a9cde0e45da is done we can revert this change?

cc @WangGuangxin

Reply:
ok